Simplenote/CrashLogging.swift

```diff
@@ -110,6 +107,7 @@ struct CrashLoggingCache {
         }
     }
 
+    private(set)
```

Comment:
You can remove the `fileprivate` too. 😉 

```swift 
    private(set) static var cachedUser: User? {
```

Reply:
Sure can! 1eb9d81d7162245a7bbcad587bd57a7609c97f3b